src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -201,323 +147,8 @@ public void assigner_has_grant_type_specific_and_permission_manage_own_can_assig
-    public void assigner_has_grant_type_challenged_and_permission_own_manage_should_not_assign_a_task_to_assignee_with_cancel_permission() {
-        //assigner role : own, manage
-        //assignee role : cancel
-        testGrantType = GrantType.CHALLENGED;
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "reviewSpecificAccessRequestJudiciary");
-        taskId = taskVariables.getTaskId();
-
-        common.setupChallengedAccessAdmin(assignerCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-        common.setupCFTJudicialOrganisationalRoleAssignment(caseworkerForReadCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-
-
-        initiateTask(taskVariables, caseworkerForReadCredentials.getHeaders());
-
-        common.setupChallengedAccessLegalOps(assigneeCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-
-        Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED,
-            taskVariables.getTaskId(),
-            new AssignTaskRequest(getAssigneeId(assigneeCredentials.getHeaders())),
-            assignerCredentials.getHeaders()
-        );
-
-        result.then().assertThat()
-            .statusCode(HttpStatus.FORBIDDEN.value())
-            .and()
-            .body("type", equalTo(ROLE_ASSIGNMENT_VERIFICATION_TYPE))
-            .body("title", equalTo(ROLE_ASSIGNMENT_VERIFICATION_TITLE))
-            .body("status", equalTo(403))
-            .body("detail", equalTo("Role Assignment Verification: The user being assigned the Task has failed the Role Assignment checks performed."));
-
-        common.cleanUpTask(taskId);
-    }
-
-    @Test
-    public void assigner_has_grant_type_challenged_and_permission_manage_can_assign_task() {
-        //assigner role : manage
-        //assignee role : own, manage
-        testGrantType = GrantType.CHALLENGED;
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "reviewSpecificAccessRequestJudiciary");
-        taskId = taskVariables.getTaskId();
-
-        common.setupChallengedAccessJudiciary(assignerCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-        common.setupCFTJudicialOrganisationalRoleAssignment(caseworkerForReadCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-
-        initiateTask(taskVariables, caseworkerForReadCredentials.getHeaders());
-
-        common.setupChallengedAccessAdmin(assigneeCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-        assignTaskAndValidate(taskVariables, getAssigneeId(assigneeCredentials.getHeaders()));
-
-        common.cleanUpTask(taskId);
-    }
-
-    @Test
-    public void assigner_has_grant_type_challenged_and_permission_read_manage_cancel_can_assign_task_user_has_permission_execute_manage() {
```

Comment:
can this be kept as an FT? It's a good example of multiple permissions being true for both assigner and assignee